kvmagent/kvmagent/plugins/vm_local_volume_cache/command_wrapper/filesystem.py

```
@@ -0,0 +1,263 @@
+        _tmp_file = tmp_file or ".tmp"
+        _tmp_file_path = os.path.join(directory, _tmp_file)
+        try:
+            fd = os.open(_tmp_file_path, os.O_CREAT | os.O_WRONLY | os.O_EXCL, 0o600)
+            os.close(fd)
+            os.stat(_tmp_file_path)
+            os.unlink(_tmp_file_path)
+            return True
+        except (OSError, IOError) as e:
+            try:
+                os.unlink(_tmp_file_path)
+            except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)